packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5964,6 +5968,23 @@ export default class Meeting extends StatelessWebexPlugin {
     return !this.isRoapInProgress;
   }
 
+  private forceSendStatsReport = async ({callFrom}: {callFrom?: string}) => {
+    const LOG_HEADER = `Meeting:index#forceSendStatsReport --> called from ${callFrom} : `;
+    // media failed, so collect a stats report from webrtc using the wcme connection to grab the rtc stats report
+    // send a webrtc telemetry dump to the configured server using the internal media core check metrics configured callback
+    try {
+      await this.mediaProperties?.webrtcMediaConnection?.forceRtcMetricsSend();
```

Comment:
Please add unit test, `forceRtcMetricsSend` is private but you could check if `webrtcMediaConnection?.forceRtcMetricsSend()` called or not

Reply:
I have a unit test for it

Reply:
the unit test is in meetings/index.js (the spec file) and does exactly what you said. btw this was here before, just hard to see I think.